docs/26_export_manual.md

```diff
@@ -0,0 +1,269 @@
+% Data Export
+% Colorado Risk Limiting Audit Tool 
+% 2017
+
+Separate from the RLA central server and client server programs that support the Department of State and the Counties in carrying out the Risk Limiting Audit, there is a command, called `rla_export`, allowing export of data from the central server and the underlying database. 
+
+The command is part of a python package, whose technical description can be found in a `README.rst` file in the python site-packages directory tree wherever the package is installed. The most current online version (which may or may not match the version you have installed) is available 
+in the [public code repository](https://github.com/FreeAndFair/ColoradoRLA/tree/auditcenter/server/eclipse-project/script/rla_export).
+
+The `README.rst` file gives instructions for installing the python package, and describes various run-time options. For a catalog of the exports produced by the command, see below.
+
+
+## Database Exports
+The `rla_export` command exports many of the files necessary for independent verification of 
+the RLA, whether by candidates, parties, other organizations. 
+Specific exports are detailed below.
+
+
+
+### Minimum Data Required to Allow Public to Reproduce Audit Calculations
+
+To allow independent verification of the RLA, 
+the Colorado Department of State must provide to the public all the export files listed in this section,  along with:
+
+- all CVR files
+- all ballot manifest files
+- opportunity to observe the random seed selection
+- opportunity to observe the activities of the County Audit Boards
+- announced tabulated results for contests selected for audit
+- the risk limit
+- the error inflation factor
+
+Providing the above to the public, along with the database exports listed below will fulfill the minimum
+requirements of a publicly-verifiable audit.
+
+The data files in this section are generated based on ``sql`` query files.
+These are always produced in two formats: json and csv.
+The basename of each resulting file is the same as the basename of the query file.
+Thus, given the query file ``seed.sql``, the files ``seed.json`` and ``seed.csv``
+will be produced.
+
+#### m_selected_contest_audit_details_by_cvr.sql
+
+ For each contest under audit, and for each cast vote record presented to the 
+  Audit Board for verification that contains the given contest, 
```

Comment:
=> That **affects** the current contest, since ballot-not-found ACVRs affect all the contests in the county.